bundles/WebToPrintBundle/src/Processor/Gotenberg.php

```diff
@@ -33,34 +33,34 @@ class Gotenberg extends Processor
     protected function buildPdf(PrintAbstract $document, object $config): string
     {
         $web2printConfig = Config::getWeb2PrintConfig();
-        $web2printConfig = $web2printConfig['gotenbergSettings'];
-        $web2printConfig = json_decode($web2printConfig, true);
+        $gotenbergSettings = $web2printConfig['gotenbergSettings'];
+        $gotenbergSettings = json_decode($gotenbergSettings, true);
 
         $params = ['document' => $document];
         $this->updateStatus($document->getId(), 10, 'start_html_rendering');
         $html = $document->renderDocument($params);
 
         $params['hostUrl'] = 'http://nginx:80';
```

Comment:
Should we set the nginx default for Chromium too?

Reply:
No, we shouldn't, because Chromium has the option to work with the binary or via dockerized chromium, while for Gotenberg it is mandatory to be working by Docker (due its nature of being a docker service).